### PR TITLE
Mh issue 130 baffled ammo

### PR DESCRIPTION
**Pull request by bleehu**
Part 4 of 5. Ammo for extra-stealthiness. Based on loot from the Chasing Calihan mission.

### CODE REVIEW

File: CharacterCreation/04_00_WeaponAttachments.txt [hunk 1]
```diff
@@ -431,6 +431,18 @@ Takes up twice the space in your magazine inventory as a standard magazine.
...
+== Baffled Ammo ($1.5 x mag cost) ==
+
+    This sabot-discarding ammo has baffled edges so that it barely makes any
+noise as it travels through the air. There is no zip or snap noise as bullets
+whiz by or into targets. 
+
+    Using Baffled ammo in your weapon reduces the range of your R3 bracket by
+10m and reduces the damage of your weapon by 5 damage. Stealth rolls to hide
+silent weapons fire automatically succeed. A bullet impact can be heard if a
+character is within 5m of the impact, actively curious and makes a search check
```
Comment (Turtlelord26) [inline CharacterCreation/04_00_WeaponAttachments.txt]: Search check -> Awareness save?

I would also consider 'alert' over 'actively curious'. Making it less impossible for the enemy to notice the shots at close range adds to the commando puzzle by incentivizing taking out isolated targets first.

**Review by Turtlelord26 — CHANGES_REQUESTED** [review]
Requests up for discussion, changes not necessarily required.

File: CharacterCreation/04_00_WeaponAttachments.txt (hunk 1 above)
Comment (bleehu) [reply CharacterCreation/04_00_WeaponAttachments.txt]: Agreed on both points. Update pending.

**Review by Turtlelord26 — APPROVED** [review]
Shipit

**Pull request merged**